typescript/nomad-deploy/scripts/staging/enroll.ts

```diff
@@ -56,4 +57,11 @@ const kovanDomain = deploysToSDK(
 
 const sdk = NomadContext.fromDomains([rinkebyDomain, kovanDomain]);
 
-enrollSpoke(sdk, kovanDomain.id, kovan.stagingConfig.watchers);
+(async () => {
+  await enrollSpoke(sdk, kovanDomain.id, kovan.stagingConfig.watchers);
+  await checkHubToSpokeConnection(
```

Comment:
❗  in staging, this won't work, because `enrollSpoke` won't enroll the spoke immediately. there will be latency. we need to approve and execute the gnosis tx.

Reply:
❗  also, this file is running `deployNewChain` and `deployNewChainBridge` -- would need to await those too if keeping in the same file. but probably better to split them up into diff files.

Reply:
note: right now `enrollSpoke` enrolls a bridge and a core, but in the future we may need to enroll only a core (not a bridge) if we open channels to chains where we don't deploy the bridge (e.g. if we are deploying other apps to those chains but want to avoid liquidity fragmentation)

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/58/commits/98a1ba5ede55637af6718fc97fa1227ea13c323c